scripts/test-manual-e2e.sh

```diff
@@ -24,98 +24,139 @@ info() {
     echo -e "$BLUE""$*""$ENDCOLOR"
 }
 
+kill_packagers() {
+    success "Killing any running packagers"
+    lsof -i :8081 | grep LISTEN
+    lsof -i :8081 | grep LISTEN | /usr/bin/awk '{print $2}' | xargs kill
+}
+
+REPO_ROOT=$(pwd)
+CLI_PATH="$REPO_ROOT/cli.js"
+
 PACKAGE_VERSION=$(cat package.json \
   | grep version \
   | head -1 \
   | awk -F: '{ print $2 }' \
   | sed 's/[",]//g' \
   | tr -d '[[:space:]]')
 
-success "Preparing version $PACKAGE_VERSION"
-
-repo_root=$(pwd)
-
-rm -rf android
-./gradlew :ReactAndroid:installArchives || error "Couldn't generate artifacts"
-
-success "Generated artifacts for Maven"
-
-npm install
-
-success "Killing any running packagers"
-lsof -i :8081 | grep LISTEN
-lsof -i :8081 | grep LISTEN | /usr/bin/awk '{print $2}' | xargs kill
-
-info "Start the packager in another terminal by running 'npm start' from the root"
-info "and then press any key."
-info ""
-read -n 1
-
-./gradlew :RNTester:android:app:installJscDebug || error "Couldn't build RNTester Android"
+ANDROID_EMULATOR_NAME=${ANDROID_EMULATOR_NAME:-$(emulator -list-avds | awk '{print $1}')}
 
-info "Press any key to run RNTester in an already running Android emulator/device"
-info ""
-read -n 1
-adb shell am start -n com.facebook.react.uiapp/.RNTesterActivity
+# Prepare
```

Comment:
I would probably change a bit this section to 'clean up the house' more. For ex. I'd extract (similar to what happens to `kill_packagers`) all the cleanup part to its own method and in it, something along the lines of:
```
clean_folders() {
    rm -rf android
	rm react-native-*.tgz
    rm -rf /tmp/RNTestProject
	rm -rf node_modules 
}
```

and call it after `    info "Preparing version $PACKAGE_VERSION"`

What do you think?

PS: maybe we could even add some more scripts to that like
`rm -rf ~/Library/Developer/Xcode/DerivedData/*`